recipes-support/scripts/files/aerofc-get-version.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
+import os,sys
+from pymavlink import mavutil
+
+if len(sys.argv)!=2:
+    print "Provide ipaddress:port like below,"
+    print "aerofc-get-version.py 0.0.0.0:14550"
+    sys.exit(0)
+
+mav_con = mavutil.mavlink_connection('udp:'+sys.argv[1],planner_format=False, notimestamps=True, robust_parsing=True)
```

Comment:
Please use TCP instead. Due to the way we use mavlink-router it will lock the endpoint to that IP (in this case the local one) so it won't allow connection from GCS without restarting it.  Port 14550 is dedicated for GCS.

For TCP mavlink-router creates and destroy new endpoints in runtime.